src/main/java/com/kakao/golajuma/vote/domain/service/CloseVoteService.java

```diff
@@ -0,0 +1,30 @@
+package com.kakao.golajuma.vote.domain.service;
+
+import com.kakao.golajuma.vote.domain.exception.CloseException;
+import com.kakao.golajuma.vote.domain.exception.NullException;
+import com.kakao.golajuma.vote.infra.entity.VoteEntity;
+import com.kakao.golajuma.vote.infra.repository.VoteRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Transactional(readOnly = true)
+@RequiredArgsConstructor
+@Service
+public class CloseVoteService {
+	private final VoteRepository voteRepository;
+
+	public void closeVote(long voteId, Long userId) {
+		VoteEntity vote =
+				voteRepository.findById(voteId).orElseThrow(() -> new NullException("해당 투표가 존재하지 않습니다."));
+		// 작성자가 아닌 경우 예외
+		if (!vote.isOwner(userId)) {
```

Comment:
```suggestion
		if (vote.isNotOwner(userId)) {
```

개인적으로는 메소드에 Not도 표현하는 것을 좋아합니다.